test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -88,13 +105,13 @@ public string[] GetInstallCommandsFromLocal(string path)
         {
             SupportedPackageExtension.Deb => new[]
             {
-                "apt-get purge --yes libiothsm-std iotedge"
+                "dpkg --purge libiothsm-std aziot-edge aziot-identity-service iotedge"
```

Comment:
Since `libiothsm-std` and `iotedge` pakages are never installed, we don't need to clean them up.

Reply:
They may be installed on the test machines because a previous pipeline ran against `master`.

Reply:
The cleanup from previous tests should handle it. The new tests should cleanup their own installed packages.

Reply:
Currently, the test tear down on master doesn't uninstall packages. It only deletes created devices.

I remember running into some issues when running on Pipelines from having both iotedge (from a previous run) and aziot-edge installed. Purging the packages before and after the run fixed that.

Reply:
It is a tad annoying to cleanup old packages, but I see no way out of this for now. Thanks for patiently explaining!